app/code/Magento/Directory/Model/Currency.php

```diff
@@ -431,6 +431,7 @@ private function formatCurrency(string $price, array $options): string
         if ((array_key_exists(LocaleCurrency::CURRENCY_OPTION_DISPLAY, $options)
                 && $options[LocaleCurrency::CURRENCY_OPTION_DISPLAY] === \Magento\Framework\Currency::NO_SYMBOL)) {
             $formattedCurrency = str_replace(' ', '', $formattedCurrency);
+            $formattedCurrency = str_replace(' ', '', $formattedCurrency);
```

Comment:
What is the idea here with duplicated line?

Reply:
Oh, that is mistake

Reply:
```suggestion
```